src/components/Inputs/ExchangeAmountInput/style.ts

```diff
@@ -31,13 +32,65 @@ const useStyles = makeStyles(() => ({
     marginRight: 20
   },
   input: {
-    paddingTop: 4,
-    '&:focus': {
-      color: colors.white.main
+    textAlign: 'right',
+    color: colors.white.main,
+    '& ::placeholder': {
+      textAlign: 'right'
     }
   },
   label: {
     top: -1
+  },
+  BalanceContainer: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  BalanceTypography: {
+    cursor: 'default !important',
+    color: colors.invariant.lightGrey,
+    ...newTypography.caption3,
+    paddingLeft: 15,
+    marginRight: 3
+  },
+  container: {
+    paddingRight: 15,
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+    filter: 'brightness(0.8)',
+
+    '& :hover > button , & :hover > p': {
+      filter: 'brightness(1.1)'
+    }
```

Comment:
Come on. This is BS. We use material ui so you dont have to deal with selectors.